druid/src/widget/window_drag_handle.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2018 The Druid Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! A [`Controller`] widget that makes the target acts as a window grab handle.
+//!
+//! [`Controller`]: struct.Controller.html
+
+use tracing::instrument;
+
+use crate::widget::prelude::*;
+use crate::widget::{Controller, ControllerHost};
+
+/// A [`Controller`] widget that makes the target acts as a window grab handle.
+///
+/// On most platforms, click events aren't passed through to the children. As such, it is
+/// recommended to only put text or small icons in a window drag handle.
```

Comment:
Regarding the cross-platform behavior, I think we want as many things as possible/reasonable to behave the same between platforms. It's fine to leave things stubbed out if it's too much effort to do it *for now* on other platforms, but I'd argue against most other plaform-specific stuff. (This is mainly a complaint about the docs, though, because AFAICT the implementation doesn't actually swallow the click events)